workshop/content/030_basic_content/075_serverless/_index.md

```diff
@@ -5,7 +5,264 @@ weight: 75
+
+* inject latency into serverless calls,
+* change the response code of the serverless function, and
+* simulate exceptions in code execution.
+
+A similar JS library, [**failure-lambda**](https://github.com/gunnargrosch/failure-lambda) is described in the [**Serverless Chaos workshop**](https://catalog.us-east-1.prod.workshops.aws/workshops/3015a19d-0e07-4493-9781-6c02a7626c65/en-US/serverless/failure-lambda/fault-injection) and the understanding of the priciples should allow the reader to build their own in their preferred language.
+
+## Architecture
+
+In AWS Lambda, serverless functions expose a "handler" function that receives a JSON object and returns a JSON object. We can keep this handler function unchanged by inserting a new wrapper function around the original handler function, e.g. using a [**decorator pattern**](https://en.wikipedia.org/wiki/Decorator_pattern). This wrapper can cause exceptions in lambda function execution before or after customer code is called, can inject latency before or after customer code is called, can modify output thus simulating a failure result, and can even modify input thus triggering failures in the customer function code:
+
+{{< img "serverless.png" "Serverless encapsulation" >}}
+
+In chaos-lambda, to allow injecting failures on-demand, the wrapper function will query an [**SSM Parameter Store parameter **](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) to check whether failures should be injected at all and, if so, what failures.
+
+To inject failures in the context of an FIS experiment, we will use an [**SSM Automation**](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-automation.html) document to change the value of the SSM Parameter Store parameter and turn on different types of failures.
+
+## Experiment idea
+
+In this section we are exploring tooling so we will start without a hypothesis. However, we will provide some learnings and next steps at the end.
+
+Specifically in this section we will an API backed by AWS Lambda instrumented with chaos_lambda and run an FIS experiment that will, in order:
+
+* inject latency
+* injedt an error code response
+* inject a runtime exception
+
+We will observe these changes by continuosly checking response time, response code, and response body.
+
+
+## Experiment setup
+
+{{% notice note %}}
+**The experiment setup section is for reference only. All required components have been created as part of the workshop setup. If you just want to see the serverless fault injection you can skip ahead to "Validation Procedure" below.** 
+{{% /notice %}}
+
+{{% notice note %}}
+In earlier sections we have described how confgure service roles, create FIS experiment templates, and create SSM automaton documents. For this section we have created all required resources as part of the infrastructure setup and will only outline the configuration procss on the console.
```

Comment:
how confgure service roles, -> how to configure service roles.

Reply:
For this section we have created -> For this section, we have created all THE required ..

Reply:
setup and will on  ->  setup, and will on

Reply:
procss -> process